js/src/sdk/client/types.gen.ts

```diff
@@ -408,42 +763,38 @@ export type SingleAppInfoResDTO = {
    * The metadata of the app
    */
   meta?: unknown;
-  /**
-   * The yaml of the app
-   */
-  yaml?: unknown;
 };
```

Comment:
The `yaml` field has been removed from `SingleAppInfoResDTO` but there's no deprecation notice or migration guide. If this is a breaking change, consider:
1. Adding a deprecation cycle first
2. Providing migration instructions
3. Documenting alternative ways to access this information